src/app/shared/components/city-autocomplete/city-autocomplete.component.ts

```diff
@@ -90,5 +96,12 @@ export class CityAutocompleteComponent implements OnInit, OnDestroy {
         });
     }
   }
+  
+  /**
+   * This method sets focus on input search city
+   */
+  onFocus(): void {  
```

Comment:
onFocus says that something shuold be handled on focus event. I believe setFocus is more preferable in this case